app/body/tests/test_models.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase
+
+from body.models import Muscle, MuscleGroup
+
+
+class TestBodyModels(TestCase):
```

Comment:
usually we don't bother with tests like these, it's testing the framework's behaviour
they are kinda too basic/redundant but for learning purposes i guess you may write sth like that